src/components/ec-full-screen-overlay/ec-full-screen-overlay.vue

```diff
@@ -67,6 +70,14 @@ defineProps({
     type: Boolean,
     default: false,
   },
+  showCloseBtn: {
+    type: Boolean,
+    default: true,
+  },
+  backgroundType: {
```

Comment:
should we name it backgroundColor?

Reply:
... or even `backgroundColorLevel`. The docs call them levels: https://chameleon-ebury.vercel.app/?path=/docs/css-colors--all.